### ISSUE TITLE
[Meta] Async straw polls

### ISSUE THREAD

**Issue by LeaVerou** [w3c/csswg-drafts#9438]
A common way we resolve certain issues where there are several distinct alternatives is by doing a straw poll.

During a telcon, the straw poll is done by assigning each of the options a letter or number, describing it clearly, and having people type their answer into IRC. For example, [here's a recent straw poll on the `contrast-color()` MVP](https://lists.w3.org/Archives/Public/www-style/2023Sep/0008.html#:~:text=What%20about%20a-,straw%20poll,-%3F%0A%20%20Rossen%3A%20What%20are).

However, it is unclear how we can do straw polls async. This would not only be useful for resolving on things, but even just as a quick way to see where everybody stands on an issue, which would be incredibly useful especially on bikeshedding issues.

One way is the awkward "For Option 1 use a 👍🏼 react, for Option 2 a ❤️ react" etc, but that's hard to parse, disconnected from the actual options, and there is no way to distinguish WG member votes from votes by people who are passing by.

Since every WG member has permissions to edit everyone else's comments, I propose the following:

One member posts the straw poll, e.g. for #9102 it could look like this:

> ### Straw poll
> Please edit this comment, add your at-username next to the option you support and update the count next to it. If the option is not in the list, please add it.
> 
> 1. Abstain (0): 
> 2. `text-wrap-allow` (0): 
> 3. `text-wrap-allowed` (0): 
> 4. `text-wrap-mode` (0): 
> 5. `white-space-wrap` (1): @LeaVerou

Ideally, the heading could be a link to a wiki entry that explains this process.

I worried a bit about editing conflicts (i.e. someone else edited a comment after you clicked "Edit" and before you submitted) but it appears they are not destructive:

<img width="834" alt="image" src="https://github.com/w3c/csswg-drafts/assets/175836/7b69d827-cfc7-4360-ad1a-155106906e24">

Sure, it would be annoying if everybody is trying to vote at once, but at least it's not destructive.

**Comment by fantasai** [comment]
And maybe as an escape hatch, if someone doesn't have write permissions because of some admin error, that person can add a comment asking someone to edit their response in.

**Comment by nickmccurdy** [comment]
How about GitHub Discussions with voting restricted to WG members?

**Comment by LeaVerou** [comment]
> How about GitHub Discussions with voting restricted to WG members?

If that's possible, that's a great idea!

**Comment by frivoal** [comment]
We should also include a link to all the open straw polls in emails announcing the agenda for the next synchronous meeting, to help make sure people notice them.

Not that we necessarily need to discuss them in the synchronous meeting, but just as a convenient place to inform/remind people of what homework they're supposed to do.

**Comment by LeaVerou** [comment]
@nickmccurdy 
> How about GitHub Discussions with voting restricted to WG members?

If it's possible to restrict to WG members, and it's possible to see who voted, that's a great idea! From a quick search, it appears that voters are not visible?

**Comment by nickmccurdy** [comment]
> @nickmccurdy
> 
> > How about GitHub Discussions with voting restricted to WG members?
> 
> If it's possible to restrict to WG members, and it's possible to see who voted, that's a great idea! From a quick search, [it appears that voters are not visible](https://github.com/orgs/community/discussions/17367)? That would rule this out I’m afraid.

@LeaVerou Ah yes, forgot about that. 😦 If you need to use GitHub you could use 👍 and 👎 reactions. The UI appears to truncate them, but the data may still be accessible through the API. You can also do similar reaction based polls on Discord, but it always lets you see all reactions.

**Comment by LeaVerou** [comment]
> > @nickmccurdy
> > > How about GitHub Discussions with voting restricted to WG members?
> > 
> > 
> > If it's possible to restrict to WG members, and it's possible to see who voted, that's a great idea! From a quick search, [it appears that voters are not visible](https://github.com/orgs/community/discussions/17367)? That would rule this out I’m afraid.
> 
> @LeaVerou Ah yes, forgot about that. 😦 If you need to use GitHub you could use 👍 and 👎 reactions. The UI appears to truncate them, but the data may still be accessible through the API. You can also do similar reaction based polls on Discord, but it always lets you see all reactions.

No way to restrict reactions to members, though we do use them for non-member voting (see the second such poll). We do not use Discord, so adding another tool to the mix just for polls seems suboptimal.
